nistats/first_level_model.py

```diff
@@ -583,6 +585,44 @@ def compute_contrast(self, contrast_def, stat_type=None,
 
         return outputs if output_type == 'all' else output
 
+    def get_voxelwise_model_attribute_(self, attribute, timeseries=True):
+
```

Comment:
There are too many blank lines in the function, which makes it difficult to read. Remove them.